tests/cases/docker/prettier/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:current
+RUN npm i -g yarn --force
+RUN git clone https://github.com/prettier/prettier.git /prettier
+WORKDIR /prettier
+RUN git pull
+COPY --from=typescript/typescript /typescript/typescript-*.tgz /typescript.tgz
+RUN mkdir /typescript
+RUN tar -xzvf /typescript.tgz -C /typescript
+RUN yarn add typescript@/typescript.tgz
+RUN yarn
+ENTRYPOINT [ "yarn" ]
+# Build
+CMD [ "build" ]
```

Comment:
Hm, so the user test previously just applied a custom tsconfig to the contents of the `src` directory and built it. That actually includes way more than they typecheck in their own build: https://github.com/prettier/prettier/blob/master/tsconfig.json Do we want to continue checking the whole src folder under our own rules like the `user` test used to, or should we use their build like this?

Reply:
I'd probably use their build, since that's what they're running.